docs/topics/dokka-migration.md

```diff
@@ -156,13 +157,15 @@ options according to your project setup:
   Previous configuration:
   
     ```kotlin
-    remoteUrl.set(URL("https://github.com/your-repo"))   
+    remoteUrl.set(URL("https://github.com/your-repo"))
     ```
   
   New configuration:
   
     ```kotlin
-    remoteUrl.set(URI("https://github.com/your-repo"))  
+    remoteUrl.set(URI("https://github.com/your-repo"))
+    // OR
+    remoteUrl("https://github.com/your-repo")
```

Comment:
DGPv2 has a couple of utility functions for setting the URL.

https://github.com/Kotlin/dokka/blob/220922378e6c68eb148fda2ec80528a1b81478c9/dokka-runners/dokka-gradle-plugin/src/main/kotlin/engine/parameters/DokkaSourceLinkSpec.kt#L82-L96